tests/Copy-DbaLogin.Tests.ps1

```diff
@@ -91,8 +91,6 @@ Describe $CommandName -Tag IntegrationTests {
             $null = Invoke-DbaQuery -SqlInstance $instance -Database tempdb -Query $dropTableQuery
         }
 
-        $null = Remove-DbaLogin -SqlInstance $TestConfig.instance1, $TestConfig.instance2 -Login 'claudio', 'port', 'tester'
```

Comment:
are they left behind?

Reply:
No. `Initialize-TestLogin` removes all the test logins. So there are no more logins to removeand so this command only outputs a warning about not finding those logins. I capture all the warnings while running the tests so I found this and removed all the code that does no work but only outputs warnings.